python/pfs/pipe2d/weekly/test_weekly.py

```diff
@@ -20,7 +20,7 @@
 @classParameters(configuration=("brn", "bmn"))
 class ProductionTestCase(lsst.utils.tests.TestCase):
     def setUp(self):
-        self.butler = Butler(os.path.join(weeklyRerun, "pipeline", self.configuration))
+        self.butler = Butler(os.path.join(weeklyRerun, "pipeline", self.configuration, "pipeline"))
```

Comment:
Why the extra "pipeline" path element? And I don't see anything in `process_weekly.sh` with a rerun ending in `pipeline`.

Reply:
When `generateCommands.py` runs a pipeline, it appends "pipeline" to the given rerun name, as well as it appends "bias", "dark", etc on constructing calibs. If this suffix "pipeline" is undesirable, I have to revise `generateCommands.py`.

Reply:
I'm not fond of this (I specify a rerun when I run `generateCommands.py`, so that's where I expect it to be), but when I thought about it I saw why this is necessary (because we want the different calibs and the science pipeline outputs separated). So ignore me and carry on.